lib/Jira.js

```diff
@@ -65,8 +65,8 @@ class Jira {
             const branchName = this.jiraSettings.branchName
             const self = this
 
-            // loop for each env.
-            for (const [jiraTargetEnv, reviewer] of REVIEWER_ENV_MAP) {
+            // loop for each env using ARRAY.
+            for (let i = 0; i < JIRA_ENV_REVIEWER.lenght; i++) {
```

Comment:
Is the index needed? If not, a for-of loop eliminates the i variable and makes typos less likely. Case in point: .lenght -> .length

Reply:
Corrected